bitswap/client/internal/session/session.go

```diff
@@ -406,6 +406,7 @@ func (s *Session) findMorePeers(ctx context.Context, c cid.Cid) {
 		for p := range s.providerFinder.FindProvidersAsync(ctx, k, 0) {
 			// When a provider indicates that it has a cid, it's equivalent to
 			// the providing peer sending a HAVE
+			log.Infof("FoundPeer %s for CID %s", p, k)
```

Comment:
Easier to search/filter for consistent string "Found peer for CID":
```suggestion
			log.Infow("Found peer for CID", "peer", p, "cid", k)
```